### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by danicatt**
Added project section and contact me section

### CODE REVIEW

File: task-header.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			<div class="infobar">
+				<a href="aboutme"> About Me </a>
+				<a href="portfolio"> Portfolio </a>
+				<a href="contactme"> Contact Me </a>
+			</div>
+			
+			<div class="social">
+				<a href="https://uk.linkedin.com/"><img src="LinkedinLogo.png" img class="socialicons"></a>
+				<a href="https://twitter.com/?lang=en-gb"><img src="twitter.png" img class="socialicons"></a>
+				<a href="https://www.instagram.com/"><img src="instagram.png" img class="socialicons"></a>
```
Comment (samcrudge) [inline task-header.html]: indenting img might be easier?
```suggestion
				<a href="https://uk.linkedin.com/">
				             <img src="LinkedinLogo.png" img class="socialicons">
				</a>
				<a href="https://twitter.com/?lang=en-gb">
				         <img src="twitter.png" img class="socialicons">
				</a>
				<a href="https://www.instagram.com/">
				            <img src="instagram.png" img class="socialicons">
				</a>
```

File: task-header.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		
+		<header class="mainintro">
+			<div class="maintitle">
+				<h1>Danielle Catterill</h1>
+				<h2>Tea Drinker, Dog Lover & Full Stack Developer in Training</h2>
+			</div>	
+		</header>
+		
+		<section class="aboutme">
+			<div><img src="smiley-2055681_1920.jpg" img class="aboutmepic"></div>
```
Comment (samcrudge) [inline task-header.html:30]: indentation mybe?

File: task-header.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+				<div class="project">
+					<div><img src="pineapple.jpg" img class="projectimage"/></div>
+					<a href="#"> Project 2 </a>
+					<a href="#"> Git Hub </a>
+				</div>
+			
+				<div class="project">
+					<div><img src="pineapple.jpg" img class="projectimage"/></div>
+					<a href="#"> Project 3 </a>
+					<a href="#"> Git Hub </a>
```
Comment (samcrudge) [inline task-header.html:58]: img indentation again possibly?

**Review by samcrudge — CHANGES_REQUESTED** [review]

File: task-header.html (hunk 1 above)
Comment (danicatt) [reply task-header.html]: Yep, agree with that.

**Pull request merged**